Analysis/Tasks/PWGHF/qaTaskSimple.cxx

```diff
@@ -155,28 +155,59 @@ struct QAGlobalObservables {
 
 /// Task to QA the kinematic properties of the tracks
 struct QATrackingKine {
-  o2fw::Configurable<int> nBinsPt{"nBinsPt", 100, "Number of bins for Pt"};
-  std::array<double, 2> ptRange = {0, 10.};
-  std::array<double, 2> invptRange = {-1.e2, 1.e2};
-  o2fw::Configurable<int> nBinsEta{"nBinsEta", 100, "Number of bins for the eta histogram."};
-  std::array<double, 2> etaRange = {-6, 6};
-  o2fw::Configurable<int> nBinsPhi{"nBinsPhi", 100, "Number of bins for Phi"};
+  o2fw::Configurable<int> pdgCode{"pdgCode", 0, "PDG code of the particle to select in absolute value, 0 selects every particle"};
+
+  o2fw::Configurable<float> etaMin{"eta-min", -3.f, "Lower limit in eta"};
+  o2fw::Configurable<float> etaMax{"eta-max", 3.f, "Upper limit in eta"};
+  o2fw::Configurable<float> phiMin{"phi-min", 0.f, "Lower limit in phi"};
+  o2fw::Configurable<float> phiMax{"phi-max", 2.f * M_PI, "Upper limit in phi"};
```

Comment:
> `M_PI` is not a part of the standard.

That's what I learned from some googling.
Maybe that's why there are `o2::constants::math::TwoPI` and `o2::constants::math::PI`.

Reply:
Fixed